src/esss_fix_format/cli.py

```diff
@@ -304,9 +352,22 @@ def _main(files_or_directories, check, stdin, commit, format_code):
                     files.extend(os.path.join(root, n) for n in names if should_format(n))
             else:
                 files.append(file_or_dir)
-    changed_files = []
+
+    files = [Path(x) for x in files]
     errors = []
+
+    black_config = find_black_config(files)
+    formatted_by_black = False
+    if black_config:
+        # skip pydevf formatter
+        format_code = None
```

Comment:
Wouldn't be better to have this variable renamed to `pydevf_format` or something alike?